pkg/volume/aws_ebs/aws_ebs_test.go

```diff
@@ -154,7 +154,7 @@ func TestPlugin(t *testing.T) {
 		t.Errorf("Got unexpected path: %s", path)
 	}
 
-	if err := builder.SetUp(); err != nil {
+	if err := builder.SetUp(nil); err != nil {
```

Comment:
So it looks like that would not be possible to do. The problem is that you cannot chown as non-root. This can be unit tested though by injecting chmodRunner, chownRunner but I think that should be a separate PR as this one is pretty large already.